resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -483,6 +483,7 @@ rds.validation.setup_guide=See the RDS guide for IAM authentication
 rds.validation.username=A non-empty username is required
 redshift.auth.aws=AWS Auth
 redshift.cluster_id=Cluster ID
+redshift.connect_aws_credentials=Connect With AWS Credentials
```

Comment:
don't think "with" needs to be title cased.

Reply:
Makes sense, changed